src/main.ts

```diff
@@ -43,6 +44,12 @@ const init = async (): Promise<void> => {
     const app: Express = express();
     const knexConnection = knex(config.knex);
 
+    const shutDown = async (server: Server): Promise<void> => {
+        await knexConnection.destroy();
+        server.close(() => logger.info(`server closed`));
+        process.exit();
```

Comment:
@norisalsaadie Why are we forcing an exit? surely if all the handles have been closed the process will shutdown gracefully?